gax-java/gax/src/main/java/com/google/api/gax/rpc/ApiClientHeaderProvider.java

```diff
@@ -57,6 +62,7 @@ protected ApiClientHeaderProvider(Builder builder) {
       appendToken(apiClientHeaderValue, builder.getGeneratedLibToken());
       appendToken(apiClientHeaderValue, builder.getGeneratedRuntimeToken());
       appendToken(apiClientHeaderValue, builder.getTransportToken());
+      appendToken(apiClientHeaderValue, builder.protobufRuntimeToken);
```

Comment:
Can we create + use a getter for this token value - like the others? Following the existing pattern allows future work to interact with this feature the same as the others (for example: if we decide to override the getter for testing or future features).